gno.land/cmd/gnoland/config_get.go

```diff
@@ -3,19 +3,24 @@ package main
 import (
 	"context"
 	"errors"
+	"flag"
 	"fmt"
-	"reflect"
-	"strings"
 
 	"github.com/gnolang/gno/tm2/pkg/bft/config"
 	"github.com/gnolang/gno/tm2/pkg/commands"
 )
 
 var errInvalidConfigGetArgs = errors.New("invalid number of config get arguments provided")
 
+type configGetCfg struct {
+	configCfg
+
+	raw bool
+}
+
 // newConfigGetCmd creates the config get command
 func newConfigGetCmd(io commands.IO) *commands.Command {
-	cfg := &configCfg{}
+	cfg := &configGetCfg{}
 
 	cmd := commands.NewCommand(
 		commands.Metadata{
```

Comment:
Change usage to `config get [<key>]`

Reply:
Updated:

[4b6b337](https://github.com/gnolang/gno/pull/2393/commits/4b6b3379dca114e7b41502b9c0436e315a01676e)